gems/laa_multi_step_forms/app/validators/time_period_validator.rb

```diff
@@ -43,7 +40,36 @@ def validate_period(time_period)
       add_error(:invalid)
     end
   end
-  # rubocop:enable Metrics/AbcSize
+
+  def validate_hours(hours)
+    add_error(:blank_hours) if hours.nil?
```

Comment:
are we sure this is a nil and not an empty string? i.e. would this be better with `hours.blank?`

Reply:
Interestingly Rails Magic (TM) means that by the time the value makes it to our custom attribute type, a blank string has been converted into a nil. But since it's magic I think you're right, better to check for blanks since blanks are explicitly what we're interested in. I've updated this.